src/components/AnonymousChat/ChatDetailHeader.tsx

```diff
@@ -72,8 +72,8 @@ const ChatDetailHeader = ({
   onBackPress = () => console.log('onBackPress'),
   onThreeDotPress = () => console.log('onThreeDotPress'),
   onAvatarPress = () => console.log('onAvatarPress'),
-  anon_user_info_emoji_code,
-  anon_user_info_color_code,
+  anon_user_info_emoji_code = null,
+  anon_user_info_color_code = null,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The default values for `anon_user_info_emoji_code` and `anon_user_info_color_code` are set to `null`. This could potentially lead to null pointer exceptions if these props are used without checking for null. Consider setting a default value or ensure null checks are performed before using these props.
```diff
-  anon_user_info_emoji_code = null,
-  anon_user_info_color_code = null,
+  anon_user_info_emoji_code = 'default_emoji_code',
+  anon_user_info_color_code = 'default_color_code',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
there is no default emoji